lightbeam/src/tests.rs

```diff
@@ -1,8 +1,7 @@
 use super::{module::ExecutionError, translate, ExecutableModule};
-use wabt;
 
 fn translate_wat(wat: &str) -> ExecutableModule {
-    let wasm = wabt::wat2wasm(wat).unwrap();
+    let wasm = wast::parse_str(wat).unwrap();
```

Comment:
Heh it's never too late to rename things when they're just published :)

I ideally wanted the `wat` and `wast` crates where `wat` would be a stable "just parse a thing to bytes" and `wast` would be the full AST which has lots of breaking changes as it evolves over time.

I actually just pinged the current owner of the `wat` crate since that hasn't seen activity in 3+ years, so we'll see where that goes. I do agree though `wat-parser` is probably a better name than `wast` since wast != wat. I just wanted a short name for the main crate!

I'll probably wait a day or so to see if the `wat` owner responds, and failing that renaming `wast` to `wat-parser` seems like the most reasonable course of action.

Reply:
Ok resolved now!

The main crate is `wat` with standalone stable functions and `wast` has all the fun parsing support.